docs/source/management/monitoring/setup.md

```diff
@@ -27,27 +33,103 @@ kubectl apply -n prometheus-operator --server-side -f=https://raw.githubusercont
 kubectl -n prometheus-operator rollout status --timeout=5m deployments.apps/prometheus-operator
 ```
 
-## Deploy ScyllaDBMonitoring
+## Configure `ScyllaDBMonitoring`
+
+You can see an example of a complete `ScyllaDBMonitoring` in this {{'[example manifest](https://raw.githubusercontent.com/{}/{}/examples/monitoring/v1alpha1/scylladbmonitoring.yaml)'.format(repository, revision)}}. 
+
+You can make a local copy of it (e.g., under `./scylladbmonitoring.yaml`) and modify it to fit your needs. We'll go through the most important options below.
+
+### Endpoints selector
+
+This field is used to select the ScyllaDB nodes that will be monitored. In `matchLabels` you should specify the labels that match the ScyllaDB
+Services' labels that are added by {{productName}}:
+
+- `scylla-operator.scylladb.com/scylla-service-type: member` - This is a static label that's always added to each ScyllaDB node Service.
+- `scylla/cluster: <scyllacluster-name>` - This label's value depends on your ScyllaCluster name. Replace `<scyllacluster-name>` with the actual name of your ScyllaCluster.
+
+```yaml
+spec:
+  endpointsSelector:
+    matchLabels:
+      scylla-operator.scylladb.com/scylla-service-type: member # Each Scylla node Service has this label.
+      scylla/cluster: <replace-with-your-scyllacluster-name>   # Replace with your ScyllaCluster name.
+```
+
+### Prometheus configuration
+
+By default, `ScyllaDBMonitoring` will deploy a Prometheus instance for you (Managed mode). You can also change the mode to
+External if you want to use an existing Prometheus instance in your cluster.
+
+```yaml
+spec:
+  components:
+    prometheus:
+      mode: Managed # or External
+```
+
+:::::{tabs}
+::::{group-tab} Managed mode
+
+In the Managed mode, you can configure various aspects of the Prometheus instance that will be deployed for you.
+You can set resource requests and limits (under `resources`), persistency (under `storage`), and workload placement (under `placement`).
+
+::::
+
+::::{group-tab} External mode
+
+In the External mode, you need to ensure that your existing Prometheus instance is configured to discover and scrape the
+ServiceMonitor and PrometheusRule resources created by `ScyllaDBMonitoring`. This requires setting the
+`serviceMonitorSelector` and `ruleSelector` in the `Prometheus` resource to match the labels used by `ScyllaDBMonitoring`
+(or leaving them empty to select all resources).
+
+You also need to configure Grafana so that it can access the `Prometheus` instance. This will be explained in the Grafana configuration section below.
```

Comment:
This feels more like the "overview" section repeated - it says what you need to do but doesn't exactly show how. It could use some examples (e.g. similar to the yaml snippet above, or just links for two different in-tree examples?).

Reply:
Agreed. Fixed by making the overview section less specific and extending the guide with a YAML example and a note.